include/uxr/client/profile/multithread/multithread.h

```diff
@@ -32,6 +32,8 @@ struct uxrSession;
 #elif defined(PLATFORM_NAME_FREERTOS)
 #include "FreeRTOS.h"
 #include "semphr.h"
+#elif defined(PLATFORM_NAME_RPIPICO)
```

Comment:
Where is `PLATFORM_NAME_RPIPICO` defined?

Reply:
I gathered from here https://github.com/eProsima/Micro-XRCE-DDS-Client/issues/233 that `PLATFORM_NAME_FREERTOS` was defined by the user so followed the same logic. I stuck `add_compile_definitions(PLATFORM_NAME_RPIPICO)` at the bottom of  the toolchain.cmake file and it works however the its not detecting the "pico/mutex.h" so I am working on linking that.

Reply:
You will need to add the include folders of Pico SDK on the CMake toolchain.

Reply:
It looks like the problem is the code is being compiled with -std=c99 somewhere but the pico sdk needs c11 to compile with the static_assert function. I cloned the sdk repository branch and patched that and am able to build just fine.

Reply:
nvm. I Just found the `UCLIENT_C_STANDARD` argument so all I had to do was set that to 11 and it compiled fine.

Reply:
Actually adding `"-DUCLIENT_C_STANDARD=11"` to the cocon.meta solved this problem for the microxrcedds_client but i get the same error in rmw_microxrcedds where it looks like the C_STANDARD is hard coded to 99. If this [CMakeList ](https://github.com/micro-ROS/rmw_microxrcedds/blob/humble/rmw_microxrcedds_c/CMakeLists.txt) file could change that static 99 to an argument with a default of 99 like UCLIENT_C_STANDARD [here](https://github.com/eProsima/Micro-XRCE-DDS-Client/blob/master/CMakeLists.txt) i think this would not be a problem.

Reply:
Thanks, we will merge this asap